### PR TITLE
Fix for RN >= 0.40

### PR DESCRIPTION
**Pull request by scerelli**
This fix errors in `react-native`>= 0.40.

### CODE REVIEW

File: lib/ios/AirGoogleMaps/AIRGMSPolygon.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
 //
 
 #import <GoogleMaps/GoogleMaps.h>
-#import "UIView+React.h"
+#import "React/UIView+React.h"
```
Comment (christopherdro) [inline lib/ios/AirGoogleMaps/AIRGMSPolygon.h]: Should this be `<React/UIView+React.h>`?

**Comment by scerelli** [comment]
Ye, let me fix that, I've some other changes to merge into this repo so I think you can close this PR and I'll open a new one

**Comment by christopherdro** [comment]
Sounds good. Thanks for the update!

**Comment by atom-r** [comment]
I just had a failure to compile on iOS because Polyline does the wrong import, just like Polygon as shown above in this PR.

I can provide more information about my setup if needed.

**Comment by aaronbuchanan** [comment]
same as @atom-r 

As a workaround, I've added this to my package.json scripts for now:
`"postinstall": "sed -i '' 's/#import \"UIView+React.h\"/#import <React\\/UIView+React.h>/' ./node_modules/react-native-maps/lib/ios/AirGoogleMaps/AIRGMSPolyline.h"`

I feel dirty, lmk when a proper fix is up?

**Comment by aaronbuchanan** [comment]
closing my own loop here, the fix is in [e085e2](https://github.com/airbnb/react-native-maps/commit/d62f5ef5ae4edb065643dc1e675753b134e085e2), but hasn't yet made it to npm.

**Pull request closed without merging**